mobile/components/Lists/CommunityListItem.js

```diff
@@ -2,12 +2,20 @@
 import React, { Component } from 'react';
 import Avatar from '../Avatar';
 import { ListItem } from './ListItem';
-import { TextColumnContainer, Title, Subtitle, AvatarWrapper } from './style';
+import Icon from '../Icon';
+import {
+  TextColumnContainer,
+  Title,
+  Subtitle,
+  AvatarWrapper,
+  ViewForwardContainer,
+} from './style';
 import type { GetCommunityType } from '../../../shared/graphql/queries/community/getCommunity';
 
 type Props = {
   community: GetCommunityType,
   onPressHandler: Function,
+  noDivider?: boolean,
```

Comment:
Let's call this `divider` and default it to `true` instead of calling it `noDivider` and defaulting to false—no need for negative props, those are kinda confusing.

Reply:
Sounds good